### PR TITLE
Help fixes mason

### PR DESCRIPTION
**Pull request by oplambeck**
Updated how Mason Modify goes through arguments so it doesnt hit the mason.toml error before it finds the help command
Added EXECENV and SKIPIF files to mason-help-scope test file.

### CODE REVIEW

File: test/mason/mason-help-scope/EXECENV [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```
Comment (ben-albrecht) [inline test/mason/mason-help-scope/EXECENV]: Can this be a symlink to `test/mason/EXECENV`?

File: test/mason/mason-help-scope/SKIPIF [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
```
Comment (ben-albrecht) [inline test/mason/mason-help-scope/SKIPIF]: Can this be a symlink to `test/mason/SKIPIF`?

File: tools/mason/MasonModify.chpl [hunk 3]
```diff
@@ -28,46 +28,48 @@ proc masonModify(args) throws {
       masonModifyHelp();
       exit();
```
Comment (Spartee) [inline tools/mason/MasonModify.chpl]: should be exit(0);

File: test/mason/mason-help-scope/masonHelpScope.future [hunk 4]
```diff
@@ -0,0 +1 @@
+feature request: need nightly testing to build mason to run this test
```
Comment (Spartee) [inline test/mason/mason-help-scope/masonHelpScope.future]: .bad?

File: tools/mason/MasonModify.chpl (hunk 3 above)
Comment (Spartee) [inline tools/mason/MasonModify.chpl]: exit(0);

File: tools/mason/MasonModify.chpl [hunk 5]
```diff
@@ -28,46 +28,48 @@ proc masonModify(args) throws {
...
     }
+    else if args.size > 2 {
+      for arg in args[1..] {
+        if arg == '-h' || arg == '--help' {
+          masonModifyHelp();
+          exit();
```
Comment (Spartee) [inline tools/mason/MasonModify.chpl]: exit(0);

**Comment by Spartee** [comment]
Approved assuming this passed start_test after changes from review.

**Pull request merged**